findbugs/build.gradle

```diff
@@ -213,11 +130,12 @@ distributions {
 
         // TODO : These exclusions can be removed when we move dependencies from lib to dependency management
         exclude 'AppleJavaExtensions.jar'
+        exclude 'hamcrest-all-1.3.jar'
```

Comment:
who is adding hamcrest as part of the compile configuration? AFAIK it's only used as a test dependency.